collectors/CookiePopupCollector.js

```diff
@@ -0,0 +1,144 @@
+const fs = require('fs');
+
+const BaseCollector = require('./BaseCollector');
+
+// @ts-ignore
+const scrapeScript = fs.readFileSync(
+    require.resolve('./CookiePopups/scrapeScript.js'),
+    'utf8'
+);
+
+/**
+ * @param {String|Error} e
+ */
+function isIgnoredEvalError(e) {
+    // ignore evaluation errors (sometimes frames reload too fast)
+    const error = (typeof e === 'string') ? e : e.message;
+    return (
+        error.includes('No frame for given id found') ||
+        error.includes('Target closed.') ||
+        error.includes('Session closed.') ||
+        error.includes('Cannot find context with specified id')
+    );
+}
+
+class CookiePopupCollector extends BaseCollector {
```

Comment:
I see we have tests in `tests/collectors/CMPCollector.mocha.js` for `CMPCollector`. Can we add some for `CookiePopupCollector`?